plugins/infinispan100/src/main/java/org/radargun/service/ServerStartPattern.java

```diff
@@ -0,0 +1,25 @@
+package org.radargun.service;
+
+import java.util.regex.Pattern;
+
+/**
+ * Patterns which will be used into InfinispanServerLifecycle
```

Comment:
Nit: Patterns which will be used into InfinispanServerLifecycle -> Log patterns which will be used by InfinispanServerLifecycle